app/src/main/res/layout/action_bar_notification_layout.xml

```diff
@@ -25,6 +25,7 @@
         android:textColor="@color/white"
         android:layout_height="wrap_content"
         android:gravity="top|end"
+        android:visibility="gone"
```

Comment:
I have made a change here, this **TextView** was being show for a fraction of second, until we make it's visibility to **GONE** using Java 
(https://github.com/miPlodder/mifos-mobile/blob/7c93d583dbc8482a111943bdf64b32284a5194d6/app/src/main/java/org/mifos/mobilebanking/ui/fragments/HomeOldFragment.java#L289)